src/lib/common/serial.c

```diff
@@ -261,7 +266,6 @@ void serialGenericUartIRQHandler(SerialHandle_t *handle) {
 
   // Let the RTOS know if a task needs to be woken up
   portYIELD_FROM_ISR(higherPriorityTaskWoken);
-  // TODO - call this from actual irqhandler?
```

Comment:
We do not need to do this since FreeRTOS will not context switch until the ISR is fully completed.